### PR TITLE
chore: cropper area coords fix

### PR DESCRIPTION
**Pull request by RohanPunjani**

### CODE REVIEW

File: src-tauri/Cargo.toml [hunk 1]
```diff
@@ -34,7 +34,7 @@ gifski = { version = "1.12.2", default-features = false, features = [
 	"png",
 	"gifsicle",
 ] }
-helmer-media = { git = "ssh://git@github.com/helmerapp/helmer-media.git" }
+helmer-media = { git = "https://github.com/helmerapp/helmer-media.git", branch = "master" }
```
Comment (Pranav2612000) [inline src-tauri/Cargo.toml:37]: Let's keep it ssh. I'm not sure if https would work correctly for all, since `helmer-media` is a private repo

File: src-tauri/src/capturer/mod.rs [hunk 2]
```diff
@@ -8,7 +8,7 @@ use scap::{
 use tauri::{AppHandle, Event, Manager};
 use tempfile::NamedTempFile;
 
-const FRAME_TYPE: FrameType = FrameType::BGRAFrame;
+const FRAME_TYPE:scap::frame::FrameType = scap::frame::FrameType::BGRAFrame;
```
Comment (Pranav2612000) [inline src-tauri/src/capturer/mod.rs:11]: Is this change required? I think it should still work with the older code now.

File: src/pages/cropper.astro [hunk 3]
```diff
@@ -155,7 +155,7 @@ import Base from "../components/base.astro";
 	function showToolbar(buttonX: number, buttonY: number) {
 		invoke('show_toolbar', {
 			buttonCoords: [buttonX, buttonY],
-			area: [startX, startY, endX, endY]
+			area: [startX + 2, startY + 2 , endX - 7, endY - 7]
```
Comment (Pranav2612000) [inline src/pages/cropper.astro:158]: Nit: Can you write a comment on why these magic numbers were chosen?

Comment (RohanPunjani) [reply src/pages/cropper.astro:158]: Picked the numbers using some trial and errors to remove the border from preview and final gif
